src/toil/utils/toilSSHCluster.py

```diff
@@ -26,6 +26,11 @@ def main():
     parser.add_argument('-p', "--provisioner", dest='provisioner', choices=['aws'], required=True,
                         help="The provisioner for cluster auto-scaling. Only aws is currently"
                              "supported")
+    parser.add_argument('-z', '--zone', dest='zone', required=False,
```

Comment:
Didn't notice this before. Any way we can eliminate the duplication. That's a rhetorical question, of course.